.claude/scripts/release_testing/precheck_ecrc.py

```
@@ -0,0 +1,268 @@
+                "timestamp": datetime.now().isoformat(),
+            }
+            if args.json_output:
+                print(json.dumps(not_found, indent=2))
+            else:
+                print(format_text(not_found, verbose=args.verbose))
+            return
+    else:
+        logger.info("Discovering latest from 4-dev-preview stream...")
+        try:
+            release = release_controller.get_latest_dev_preview()
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)